migrations/2023-11-23-124725_update_merchant_custom_name_to_seller_name_payment_link/up.sql

```diff
@@ -0,0 +1,2 @@
+-- Your SQL goes here
+ALTER TABLE payment_link RENAME COLUMN custom_merchant_name TO seller_name;
```

Comment:
These ALTER commands are not backwards compatible. Create another column instead